radar-commons-android/src/main/java/org/radarbase/android/source/SourceService.kt

```diff
@@ -123,6 +126,30 @@ abstract class SourceService<T : BaseSourceState> : LifecycleService(), SourceSt
         sourceTypes = emptySet()
         broadcaster = LocalBroadcastManager.getInstance(this)
 
+        broadcaster.run{
```

Comment:
The intention here is good, but it's probably better put in RadarService -- only one notification should run at a time and probably the most urgent one (full) should be shown. In DataHandler you could keep a whole list of storageReceiver, one per DataCache, and if at least ONE of them becomes full, show a full notification. Otherwise, if at least ONE of them becomes partially full, show the other notification.